.github/issue_template.md

```diff
@@ -0,0 +1,35 @@
+Thank you for using our package. You can describe your issue here. If you use the following template, it will help us address your issue faster. Please add a tag to the the issue: `bug`, `enhancement`, `question` or `feature request`.
```

Comment:
1. I changed from tagging in issue title to adding a tag on github. These can be more readable, and I don't see a reason to pollute titles when there is a tagging feature in github. Is that ok? We should configure all these types of tags to the repo and remove the ones we don't use.
2. I changed [ENH] to `enhancement` but what does it mean? when should I use feature request, and when enhancement? I've also added `question` type since it is mentioned below. We should describe when to use non-obvious tags

Reply:
1) I'm not sure there is a possibility to add a tag to issue by ownself. Usually, it's done by the first person who reviews the issue. We can create more that one template, with predefined tags.
2) I'm thinking Dominik want to have something about improving current functionality.

Reply:
I think that enh == FR in our case. In GH tag is rather called label and I believe that user can add it. I would just rename tag to label and leave it as is.